lxd/instance_post.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/lxc/lxd/lxd/instance/instancetype"
 	"github.com/lxc/lxd/lxd/migration"
 	"github.com/lxc/lxd/lxd/operations"
-	"github.com/lxc/lxd/lxd/project"
+	projecthelpers "github.com/lxc/lxd/lxd/project"
```

Comment:
Per @tomponline, we should avoid doing that, instead keep importing as `project` and rename any conflicting variable to `projectName` instead.